### DISCUSSION TITLE
/reports/activity search?

### DISCUSSION THREAD

**Discussion by ShaunNeighbourSwale** [snipe/snipe-it#14480] [Q&A]
Hi All

I'm looking at grabbing a filtered activity report.  Ideally I'd like to exclude one particular admin from the results.

https://snipe-it.readme.io/reference/reportsactivity says that a search term can be specified.  Any idea of allowed search string formats?

If it's no possible to exclude an admin, I could merge reports for every other admin.  Any idea which search term would specify an admin?

Thanks!

**Comment by snipe** [comment]
I'm going to convert this into an issue, since it's really a feature request.
